common/log/loggerimpl/logger_test.go

```diff
@@ -49,8 +49,7 @@ func TestDefaultLogger(t *testing.T) {
 		outC <- buf.String()
 	}()
 
-	var zapLogger *zap.Logger
-	zapLogger = zap.NewExample()
+	zapLogger := zap.NewExample()
```

Comment:
would it cause any problems to change these to `zaptest.New(t)` instead?  then logs will be scoped to the test.

Reply:
Tried changing this - tests fail.

I think `zaptest.New(t)` is good for places when you just want to pass some logger when testing things and do no pollute output for passed tests. There are some places that could be updated to use it.

Here however, we testing the logger implementation itself.